### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by rohan-deshpande** [pqx/react-ui-tree#23]
Not exactly an *issue* but I'd love to see what kind of hit others are taking performance wise, I'm currently taking the following hit for 982 nodes

![perf](https://cloud.githubusercontent.com/assets/1079637/10989513/515586e8-849c-11e5-9989-aa6a3d538cb5.png)

It's taking quite a while as you can see. Is this to be expected?

**Comment by wangzuo** [comment]
Yes, i can imagine there would be such a performance issue since the tree kept updating and react dom keep rerendering.

**Comment by rohan-deshpande** [comment]
Yeah, this is on initial load though. I guess there's no way around it ... I will probably have to implement a way to move nodes through selection rather than drag and drop.

**Comment by edgesoft** [comment]
On initial load is solved via lazy loading. I don't think @wangzuo has added this feature but still it works. I'm using it myself.
